modules/ROOT/pages/index.adoc

```diff
@@ -48,11 +49,9 @@ image::os-use-object-store-v2.png[Click Use Object Store V2 in the Runtime Manag
 * The Object Store Connector works for both Object Store version 1 and 2.
 * Object Store v2 enforces a maximum data persistence of 30 days from time of creation.
 * Accounts that have not purchased the Object Store v2 premium SKU are rate limited to 10 transactions per second per application. To go beyond the base 10 TPS rate limit, contact your MuleSoft account team.
-* In Mule 4, Object Store v2 overrides the `_defaultPersistentObjectStore` object store. No further configuration is required.
-If the connector or component that you are using uses that object store, then you have access to Object Store v2.
+* For cloud applications using Mule 4, Object Store v2 overrides the `_defaultPersistentObjectStore` object store. No further configuration is required. If a cloud connector or component uses that object store, then you have access to Object Store v2.
```

Comment:
`If a cloud connector or component uses that object store, then you have access to Object Store v2.` should be `If a cloud connector or component uses that object store, then the data will be persisted in Object Store v2.`